src/plugins/navigation/public/top_nav_menu/top_nav_menu.tsx

```diff
@@ -65,14 +66,15 @@ export function TopNavMenu(props: TopNavMenuProps) {
   }
 
   function renderLayout() {
+    const className = props.noPadding ? undefined : 'kbnTopNavMenu';
```

Comment:
Why not pass in a class name from the caller directly?
I'm sure there will be other use cases for that.